scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -78,16 +78,22 @@
         gpu-models (->> node-name->capacity vals (map :gpus) (apply merge) keys set)
         ; The following variables are only being used setting counters for monitor
         gpu-model->total-capacity (total-gpu-resource node-name->capacity)
-        gpu-model->total-consumed (total-gpu-resource node-name->consumed)]
-
-    (log/info "In" compute-cluster-name "compute cluster, all node names:" (keys node-name->node))
```

Comment:
This, or something like it is also useful for ground truth.

Reply:
Can you add a list of all nodes at one node per line in the node watch callback initialization to compensate with this being removed here?

Reply:
Done